packages/core/src/ts-patch/install-patch.ts

```diff
@@ -0,0 +1,19 @@
+import fs from "node:fs";
+import { createRequire } from "node:module";
+
+import { transformTscContent } from "./shared.js";
+
+const require = createRequire(import.meta.url);
+const typescriptPath = require.resolve("typescript");
+
+const origReadFileSync = fs.readFileSync;
+// @ts-expect-error - approach used by Volar.js
```

Comment:
[Naming] Nit: Volar.js doing this isn't why we do it; it's just the only way that works. And the root TS complaint isn't even a correct one!

```suggestion
// @ts-expect-error - TypeScript doesn't understand that the overloads do match up. 
```